src/kv/kvtypes.h

```diff
@@ -14,9 +14,10 @@
 
 namespace kv
 {
-  using Index = int64_t;
+  using SeqNo = int64_t;
   using Version = int64_t;
   using Term = uint64_t;
+  using View = uint64_t;
```

Comment:
Is this used elsewhere, or can we move it into `Consensus`?

If we can write one-line descriptions for how each of these types is used, I think it would really help.